internal/manifests/console/build.go

```diff
@@ -0,0 +1,44 @@
+package console
+
+import (
+	consolev1 "github.com/openshift/api/console/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewConsoleLink returns a new openshift api console link
+func NewConsoleLink(name, href, text, section string) *consolev1.ConsoleLink {
```

Comment:
why is this not in the `consolelink.go`?

Reply:
Same as [reply](https://github.com/openshift/elasticsearch-operator/pull/735#discussion_r655862449)